src/modules/ekf2/EKF/ekf.h

```diff
@@ -472,16 +472,26 @@ class Ekf final : public EstimatorInterface
 	{
 		clearInhibitedStateKalmanGains(K);
 
+#if false
+		// Matrix implementation of the Joseph stabilized covariance update
+		// This is extremely expensive to compute. Use for debugging purposes only.
+		auto A = matrix::eye<float, State::size>();
+		A -= K.multiplyByTranspose(H);
+		P_true = A * P_true;
```

Comment:
error: ‘P_true’ was not declared in this scope

Do you want to change this to use P directly or instead compare P_true with efficient P below?

Reply:
I used it for both: first test if using the matrix version solved the issue and then compared it. I think the most useful thing is if we can choose between the matrix and the optimized version.